client-admin/.eslintrc.yml

```diff
@@ -10,7 +11,20 @@ extends:
   - eslint:recommended
   - plugin:react/recommended
   - prettier/react
-rules: {}
+rules: #{}
```

Comment:
some of these rules I would think should be enabled.
are they disabled only to keep the linter happy and not too disruptive temporarily? If so, I'd like to see a comment along the lines of "TODO: enable these rules and fix the remaining violations"

Reply:
Oh sorry, yes, they're only there to get a clean run, and I'm removing rules as I deal with them (anyone else welcome to push commits too)

Reply:
Got through most of em, so resolving this!